### PR TITLE
Frontend locations - Map with adjustable radius

### PR DESCRIPTION
**Pull request by lucaslcode**
First commit adds two components to storybook. Please review!

Mobile emulation in Edge wasn't allowing me to change the radius. I have to try on my phone somehow.

### CODE REVIEW

File: app/frontend/src/components/EditLocationMap.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+export default function EditLocationMap({
+  user,
+  setLocation,
+  ...otherProps
+}: EditLocationMapProps) {
+  const classes = useStyles();
+
+  const center = useMemo(() => new LatLng(user.lat, user.lng), [user]);
+  //What type is this useRef supposed to be!? Marker doesn't want to work
+  const markerRef = useRef<any>(null);
```
Comment (lucaslcode) [inline app/frontend/src/components/EditLocationMap.tsx]: Couldn't get typing to work here

**Comment by lucaslcode** [comment]
Also, I get lines between the tiles, not sure why. 
image

File: app/frontend/src/components/Map.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <p>A default sized Map.</p>
+    <Map {...args}>
+      <Marker position={defaultCenter} />
+    </Map>
+    <p>A Map with a marker.</p>
+    <Map style={{ width: "400px", height: "50px" }} {...args} />
+    <p>An explicitly sized Map.</p>
+    <div style={{ width: "400px", border: "1px solid black" }}>
+      <Map {...args} grow />
+    </div>
```
Comment (amdwit) [inline app/frontend/src/components/Map.stories.tsx]: I do not think it is necessary to include all these possibilities. If it covers the use cases as in the design and the website it is ok. I think you copied this from the Avatar where these different uses grew naturally based on how we want to use it. But I am not saying I would like to have some deleted here.

File: app/frontend/src/components/EditLocationMap.tsx (hunk 1 above)
Comment (amdwit) [reply app/frontend/src/components/EditLocationMap.tsx]: Me neither. I think the correct typing is `typeof Marker` but then in next lines `getLatLng` is not accepted by Typescript.

**Review by amdwit — DISMISSED** [review]

**Comment by amdwit** [comment]
> Also, I get lines between the tiles, not sure why.
> image

I do not see those on my browsers even not on Edge for Mac. Maybe it is browser dependent. Maybe it has to do with pixel calculation rounding.

**Comment by lucaslcode** [comment]
I will split this into multiple PRs as this is already a monster. So far just adds the map with an adjustable radius to storybook.
I have pinned mapbox-gl to <2.0.0 as v2 requires an api key (even though we are indeed using one, for now).

PS. look at the commit history if you want to feel my pain

**Comment by amdwit** [comment]
I do not see a marker in Storybook/Components/Map/Primary top map

**Comment by lucaslcode** [comment]
@amdwit sorry markers got lost in refactoring and I haven't added any back in yet

**Review by amdwit — DISMISSED** [review]

**Review by amdwit — APPROVED** [review]

**Pull request merged**